systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/PodUtils.java

```diff
@@ -166,7 +166,15 @@ public static void waitUntilPodContainersCount(String podNamePrefix, int numberO
         TestUtils.waitFor("Waiting till pod" + podNamePrefix + " will have " + numberOfContainers + " containers",
             Constants.GLOBAL_POLL_INTERVAL, Constants.GLOBAL_STATUS_TIMEOUT,
             () -> kubeClient().listPodsByPrefixInName(podNamePrefix).get(0).getSpec().getContainers().size() == numberOfContainers);
-        LOGGER.info("Waiting till pod {} will have {} containers", podNamePrefix, numberOfContainers);
+        LOGGER.info("Pod {} has {} containers", podNamePrefix, numberOfContainers);
+    }
+
+    public static void waitUntilPodReplicasCount(String podNamePrefix, int exceptedPods) {
```

Comment:
It's not `PodReplicasCount` it's just `ContainerInPodCount`, better to use `waitForPodsReady()` I think or at least name it correctly.